twindle-web/style.css

```diff
@@ -79,12 +78,8 @@ header a {
   align-items: center;
 }
 
-header nav {
-  flex: 1;
-}
-
 header nav ul {
-  display: flex;
+  display: block;
```

Comment:
Here too you removed flex to block?

Reply:
afterwards i have made that code comment because making this code responsive was creating many difficulties